bin/init.bash

```diff
@@ -117,32 +117,32 @@ set_storage_class() {
           storage_class=cinder-storage
           es_storage_class=cinder-storage
 
-          [ "$(yq read "$file" 'storageClasses.cinder.enabled')" = "null" ] &&
+          [ "$(yq read "$file" 'storageClasses.cinder.enabled')" != "true" ] &&
             yq write --inplace "$file" 'storageClasses.cinder.enabled' true
           ;;
 
         exoscale)
           storage_class=nfs-client
           es_storage_class=local-storage
 
-          [ "$(yq read "$file" 'storageClasses.nfs.enabled')" = "null" ] &&
+          [ "$(yq read "$file" 'storageClasses.nfs.enabled')" != "true" ] &&
             yq write --inplace "$file" 'storageClasses.nfs.enabled' true
```

Comment:
I'm not a fan of this because we should only set this if it is unset so that a user is free to do what ever with it.
Now if a user isn't happy with `true` and sets it to `false`, running `ck8s init` again will overwrite the users value, which in my opinion isn't desirable.

Whats opinion on this @lentzi90 @Xartos ?

Reply:
The problem is that you cant use "null" as the linter fails on that. It is supposed to be a boolean value. I had it as [ "$(yq read "$file" 'storageClasses.nfs.enabled')" = "false" ] but after a conversation with  @Xartos I changed it to what it is now. Neither way is perfect.

Reply:
Ooh, I totally misinterpreted that, I thought this was a pipeline init thing. But I totally agree with you!
But why do we need this at all? Now that we default it to false instead of null

Reply:
I guess we could just remove it? I'm also not a fan of settings "magically" being changed without the user knowing about it.

Reply:
> Ooh, I totally misinterpreted that, I thought this was a pipeline init thing. But I totally agree with you!
> But why do we need this at all? Now that we default it to false instead of null

We don't but we seem to like to set nice defaults for users depending on the cloud provider that they are running on.

Reply:
Right, is this the reason to why they were defaulted to `null` to indicate that nothing was set. So that it shouldn't reset to default the next time you run `init`

Reply:
Yep. And with my solution proposed in the first comment, no field would be null after init, and they would not be touched upon consecutive inits :)

Reply:
Yes, I think that's a nice solution.
Maybe we can add some comments describing why they are set to `null` to minimize confusion if you're reading the config file directly. Like:
```bash
# These are set to null before the first init to be able to set defaults only once
```

Reply:
Implemented according to @OlleLarsson suggestion